package.json

```diff
@@ -9,7 +9,11 @@
     "_metalsmith": "bin/_metalsmith"
   },
   "scripts": {
-    "test": "make test"
+    "test": "make test",
+    "build": "make build"
+  },
+  "engines": {
+    "node": ">=4.0.0"
```

Comment:
Yeah, if you want to go ahead and add Node 0.10 to the Travis build, and if the tests pass you can make that the minimum Node version.